src/config.example.hpp

```diff
@@ -80,10 +95,13 @@ static void configure_msg_types(RosClientNode& cn) {
       "/velodyne_2dscan", webviz_constants::lidar_2d_topic, 15);
 
   cn.register_local_msg_type<sensor_msgs::CompressedImage>(
-      "/stereo/left/image_raw/compressed", webviz_constants::left_image_topic, 10);
+      "/stereo/left/image_raw/compressed",
+      webviz_constants::left_image_topic,
+      10);
   cn.register_local_msg_type<sensor_msgs::CompressedImage>(
       "/stereo/right/image_raw/compressed",
-      webviz_constants::right_image_topic, 10);
+      webviz_constants::right_image_topic,
+      10);
```

Comment:
these are just from running `make format`, which I should create git hooks for or something